File: examples/python/cycle-pwm3.py
#!/usr/bin/env python3

import pymaa as maa
import time

x = maa.PWM(0,3)
x.enable(1);
x.period_us(20)
value= 0.0

while True:
    x.write(value)
    time.sleep(0.2)
    value = value +0.01
    if value >= 1:
        value = 0.0

File: examples/cycle-pwm3.cpp
#include <unistd.h>

#include "maa.h"

int
main ()
{
  maa::PWM pwm(0, 3);
  pwm.period_us(200);
  pwm.enable(1);

  float value = 0.0f;

  while(1) {
    value = value + 0.01f;
    pwm.write(value);
    usleep(50000);
    if (value >= 1.0f) {
        value = 0.0f;
    }
    float output = pwm.read();
  }
  return 0;
}
